CHANGELOG.adoc

```diff
@@ -41,6 +41,10 @@
 | 🐛
 | fix(tekton e2e): Refer tasks from new tekton catalog task structure
 | https://github.com/knative/client/pull/966[#966]
+
+| 🎁
+| Add support for internal plugins
+| https://github.com/knative/client/pull/880[#880]
```

Comment:
lets update the date of the release (v0.16.1) as well ?

Reply:
@rhuss : also the PR number is #902

Reply:
ah right. going to fix it now.